src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1721,6 +1728,14 @@ private void SynchronizeInternal(GraphSyncData syncData)
                 var guids = runtimeCore.ExecutedAstGuids.ToList();
                 executedAstGuids[syncData.SessionID] = guids;
                 runtimeCore.RemoveExecutedAstGuids();
+
+                // There should be a CodeBlock in CodeBlockList by now
+                if (runnerCore.CodeBlockList.Any())
```

Comment:
we can also check here is pendingUIDsForDeletion is empty

Reply:
Changed it to consider any inactive nodes too (`!x.isActive`), so now it's not possible to skip this step.